packages/web-api/src/controllers/api-controller.spec.ts

```diff
@@ -12,13 +14,18 @@ export class ApiControllerMock extends ApiController {
     public readonly logger: Logger;
     public handleRequestInvoked = false;
 
-    public constructor(public readonly context: Context) {
+    // tslint:disable-next-line: no-null-keyword
+    public constructor(public readonly context: Context, public readonly serviceConfig: ServiceConfiguration = null) {
         super();
     }
 
     public async handleRequest(): Promise<void> {
         this.handleRequestInvoked = true;
     }
+
+    public getGetRestApiConfigFunc(): () => Promise<RestApiConfig> {
+        return this.getRestApiConfig;
+    }
```

Comment:
why do you need this?

Reply:
you could just make this method return super.getRestApiConfig(); & rename this method as getRestApiConfig

Reply:
updated.